### ISSUE TITLE
Invalid HTTP method when testing sample 56 using Python

### ISSUE THREAD

**Issue by TFesmire** [microsoft/BotBuilder-Samples#3722]
## Sample information
1. Sample type: MS Teams File Upload
2. Sample language: python
3. Sample name: 56.teams-file upload

## Describe the bug
I am unable to test the sample due to the error "Bad status line 'invalid HTTP method'". I am running the bot locally and using ngrok to tunnel to azure. The ngrok error I see on the web interface is always unnexpected EOF. I can interact with the web chat on Azure and my script recognizes and posts the "invalid HTTP method" error. This occurs when testing the file in teams as well. I have included screenshots of the error as well as the ngrok web interface. Any help would be appreciated. 

## To Reproduce
1. Follow the steps on the README.md file until you get to line 50  `
## Expected behavior
I expected a card to be sent after the initial message was received by the bot giving me an upload link for a file.
## Screenshots
![Screenshot 2022-03-28 165615](https://user-images.githubusercontent.com/93405347/160493956-815037ab-2df7-42aa-acf3-06a2cc07a151.png)

![image](https://user-images.githubusercontent.com/93405347/160494510-cb3d30bb-4652-442b-ba7b-167651941f5a.png)

**Comment by anishprasad01** [comment]
Thank you for the report, I will attempt to reproduce.

**Comment by anishprasad01** [comment]
Hi @TFesmire,

I was unable to reproduce your issue. I created a new Azure Bot resource, set up an Ngrok tunnel, uploaded a Teams manifest, and I was able to connect to Python sample 56, which operated correctly. See screenshot below:

![image](https://user-images.githubusercontent.com/33945547/160884192-e1b86978-eaae-42e9-8f2f-96df375f17c8.png)

I would suggest you double check your `config.py`, your Azure messaging endpoint, permissions in Azure or Teams, and any proxies or other intermediaries that might be preventing you from reaching the bot.
